package.json

```diff
@@ -13,7 +13,8 @@
         "lint:check": "eslint . --ext .js,.ts,.json",
         "lint:ci": "eslint . --ext .js,.ts,.json",
         "migrate": "prisma migrate dev",
-        "migrate:deploy": "prisma migrate deploy"
+        "migrate:deploy": "prisma migrate deploy",
+        "start": "yarn && ts-node --transpile-only src/index.ts"
```

Comment:
1. Let's avoid calling yarn inside yarn as it does weird things
2. `start` and `dev` are pretty ambiguous. Please merge #124 first and let's revisit this